plugin/evm/vm_upgrade_bytes_test.go

```diff
@@ -189,36 +181,27 @@ func TestVMUpgradeBytesNetworkUpgrades(t *testing.T) {
 		t.Fatal(err)
 	}
 
-	// VM should not start again without proper upgrade bytes.
+	// vm should not start again if SubnetEVM upgrade is not enabled in genesisBytes or upgradeBytes
 	err = vm.Initialize(context.Background(), vm.ctx, dbManager, []byte(genesisJSONPreSubnetEVM), []byte{}, []byte{}, issuer, []*common.Fx{}, appSender)
-	assert.ErrorContains(t, err, "mismatching SubnetEVM fork block timestamp in database")
+	assert.ErrorContains(t, err, "SubnetEVM upgrade is not enabled in genesis")
 
-	// VM should not start if fork is moved back
+	// vm should not start if fork is not enabled on genesis
 	upgradeConfig.NetworkUpgrades.SubnetEVMTimestamp = big.NewInt(2)
 	upgradeBytesJSON, err = json.Marshal(upgradeConfig)
 	if err != nil {
 		t.Fatalf("could not marshal upgradeConfig to json: %s", err)
 	}
 	err = vm.Initialize(context.Background(), vm.ctx, dbManager, []byte(genesisJSONPreSubnetEVM), upgradeBytesJSON, []byte{}, issuer, []*common.Fx{}, appSender)
-	assert.ErrorContains(t, err, "mismatching SubnetEVM fork block timestamp in database")
-
-	// VM should not start if fork is moved forward
-	upgradeConfig.NetworkUpgrades.SubnetEVMTimestamp = big.NewInt(30)
-	upgradeBytesJSON, err = json.Marshal(upgradeConfig)
-	if err != nil {
-		t.Fatalf("could not marshal upgradeConfig to json: %s", err)
-	}
-	err = vm.Initialize(context.Background(), vm.ctx, dbManager, []byte(genesisJSONPreSubnetEVM), upgradeBytesJSON, []byte{}, issuer, []*common.Fx{}, appSender)
-	assert.ErrorContains(t, err, "mismatching SubnetEVM fork block timestamp in database")
+	assert.ErrorContains(t, err, "SubnetEVM upgrade is not enabled in genesis")
```

Comment:
hmm these actually testing network upgrades, and enforcing subnet-evm to be activated at genesis leaves us no other testable network upgrade. Do you think it would be possible if we add a `testMode` check to vm and if so skip checking the subnet-evm activation at genesis? cc @aaronbuchwald @darioush

Reply:
I don't think that's worth it at this time. I'd prefer to re-add these tests back when there is a future network upgrade to include.

Reply:
I'll actually revert this back to the original and comment them out. We can uncomment these tests when a new NetworkUpgrade is added